attachments.py

```diff
@@ -0,0 +1,32 @@
+
+def available_features(response):
+    return [
```

Comment:
do not write json like this. Create a map/dictionary, store data there and parse it to json. also, do not write entire thing in return statement